mitodl/mitxonline
# Adding OrderReceiptPage

#### Pre-Flight checklist




#### What are the relevant tickets?
Fix #424

#### What's this PR do?
Add an order receipts page

#### How should this be manually tested?
Make sure to set up a purchasable product, create a fullfilled oder for it, got to the order summary page `/orders/history/` and click to view one of the orders.

<img width="301" alt="Screen Shot 2022-03-07 at 9 31 44 AM" src="https://user-images.githubusercontent.com/7574259/157053730-d603e20c-e00d-486d-ac2b-8fe8bbef9869.png">
<img width="1521" alt="Screen Shot 2022-03-07 at 9 31 10 AM" src="https://user-images.githubusercontent.com/7574259/157053733-93e52878-5c49-46cd-aa72-30d41b92e07f.png">
